R/zchunk_L254.transportation_USA.R

```diff
@@ -56,19 +83,24 @@ module_gcam.usa_L254.transportation_USA_DISABLED <- function(command, ...) {
 
     all_data <- list(...)[[1]]
 
+    # Silence package notes
+    region <- supplysector <- technology <- minicam.energy.input <- year <- value <- coefficient <-
+      UCD_fuel <- UCD_sector <- UCD_technology <- calibrated.value <- coefficient <- loadFactor <-
+      size.class <- state <- tranSubsector <- tranTechnology <- calibrated.value <- stub.technology <-
+      output <- output_agg <- output_cum <- share.weight <- subs.share.weight <- share.weight.year <-
+      tech.share.weight <- calOutputValue <- energy.final.demand <- base.service <- NULL
+
     # Load required inputs
-    UCD_techs <- get_data(all_data, "energy/UCD_techs")
+    UCD_techs <- get_data(all_data, "energy/mappings/UCD_techs")
```

Comment:
It would be useful to declare what UCD stands for here and a very quick description of what the file UCD_techs contains.